lib/stages/RfiFrameDrop.cpp

```diff
@@ -136,7 +140,7 @@ void RfiFrameDrop::main_thread() {
                                  / (4.0 * sk_step * sk_step));
 
         // Lock update mutex to not allow updates being processed during this critical section
-        std::unique_lock<std::mutex> lock(update_mutex);
+        lock.lock();
```

Comment:
Why not just put this section into a block and use a lock_guard? Seems more elegant.

Reply:
Variable scope. I would have to declare them outside of the block and initialize inside.